prog/vm/host_nexus.rb

```diff
@@ -90,11 +90,16 @@ def before_run
     hop_wait_prep
   end
 
+  def os_supports_slices?(os_version)
+    os_version == "ubuntu-24.04"
```

Comment:
Did we test the slices on arm64 hosts?

Reply:
Good point.

In dev environment: yes. In prod: not yet

I'll try this on couple of arm64 machines for a while before merging the PR.